graph/chromatic_number/info.toml

```
@@ -0,0 +1,17 @@
+title = 'Chromatic Number'
+timelimit = 10.0
+forum = "https://github.com/yosupo06/library-checker-problems/issues/628"
+
+[[tests]]
+    name = "example.in"
+    number = 2
+[[tests]]
+    name = "random.cpp"
+    number = 10
+[[tests]]
+    name = "big.cpp"
+    number = 10
+
+[params]
+    MIN_N = 1
+    MAX_N = 18
```

```suggestion
    MAX_N = 20
```

O(3^N)が10sでギリギリ間に合えばいい気がするので、20行きましょう